engine/security/rootless.md

```diff
@@ -76,6 +76,18 @@ testuser:231072:65536
   `sudo modprobe overlay permit_mounts_in_userns=1`
    ([Debian-specific kernel patch, introduced in Debian 10](https://salsa.debian.org/kernel-team/linux/blob/283390e7feb21b47779b48e0c8eb0cc409d2c815/debian/patches/debian/overlayfs-permit-mounts-in-userns.patch)).
    Add the configuration to `/etc/modprobe.d` for persistence.
+  
+- Rootless docker requires version of `slirp4netns` greater than `v0.4.0`.
+  Check you have this with 
+  ```
+  $ slirp4netns --version
+  ```
+  If you do not have this run 
+  ```
+  $ echo "deb http://http.us.debian.org/debian/ testing non-free contrib main" | sudo tee -a /etc/apt/sources.list
```

Comment:
This may result in upgrading the entire OS from `stable` to `testing`? (correct me if wrong)

Reply:
Using the official static binary might be better 
https://github.com/rootless-containers/slirp4netns/releases

Reply:
Fair point. I just added a link to the release page for `slirp4netns` instead.